src/validator/Tests.java

```diff
@@ -0,0 +1,55 @@
+package validator;
```

Comment:
As I wrote in General observations point 4. Tests should be moved to src/test. The test class should not be called Tests.java, but should be divided into tests of individual classes.
After taking into consideration comments from code review, there will be even more test classes. You will be able to add many JUnit tests.